frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx

```diff
@@ -35,34 +44,62 @@ const CreatePasswordForm: React.FC<Properties> = ({
     [handleSubmit, onSubmit],
   );
 
+  const [showPassword, toggleShowPassword] = useToggle(false);
+  const [showNewPassword, toggleShowNewPassword] = useToggle(false);
+  const [showRepeatPassword, toggleShowRepeatPassword] = useToggle(false);
+
   return (
     <Modal isOpen={isOpen} onClose={onCloseModal}>
-      <div className={getValidClassNames(styles['form-wrapper'], className)}>
+      <div className={getValidClassNames(styles[inputWrapper], className)}>
         <h2 className={styles['title']}>Change Password</h2>
         <form className={styles['form-wrapper']} onSubmit={handleFormSubmit}>
-          <Input
-            control={control}
-            errors={errors}
-            label="Current Password"
-            name="password"
-            type="password"
-          />
+          <div className={styles['input-wrapper']}>
+            <Input
+              type={showPassword ? 'text' : 'password'}
+              label="Password"
+              name="password"
+              control={control}
+              errors={errors}
+            />
+            <IconButton
+              label="password"
+              icon="eye"
+              onClick={toggleShowPassword}
+              className={styles[inputIcon]}
+            />
+          </div>
 
-          <Input
-            control={control}
-            errors={errors}
-            label="New Password"
-            name="newPassword"
-            type="password"
-          />
+          <div className={styles[inputWrapper]}>
+            <Input
+              type={showNewPassword ? 'text' : 'password'}
+              label="New password"
+              name="newPassword"
+              control={control}
+              errors={errors}
+            />
+            <IconButton
+              label="New password"
+              icon="eye"
```

Comment:
I think there should be 2 icons - to describe different states - one two show password (non-crossed eye), another for hide password (crossed eye)